### PR TITLE
workflow issue solved in multitenancy

### PR DESCRIPTION
**Pull request by AbijithS-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

workflow issue in multitenancy is solved
# How has the change been tested?

# Screenshots (if applicable)

image
image

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-web/src/components/Form/Stepper.js [hunk 1]
```diff
@@ -415,10 +415,11 @@ const mapStateToProps = (state) => {
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    getAllProcesses: () => {
+    getAllProcesses: (tenantKey) => {
+      console.log("tenatn",tenantKey);
```
Comment (sreehari-aot) [inline forms-flow-web/src/components/Form/Stepper.js:419]:
```suggestion
```

**Review by auslin-aot — APPROVED** [review]

**Review by sreehari-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/Stepper.js (hunk 1 above)
Comment (abhilash-aot) [inline forms-flow-web/src/components/Form/Stepper.js:419]: Remove console

Comment (abhilash-aot) [reply forms-flow-web/src/components/Form/Stepper.js:419]: Check for Multitenancy is enabled else sent null for tenantKey

**Review by abhilash-aot — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/components/Form/Stepper.js (hunk 1 above)
Comment (AbijithS-aot) [reply forms-flow-web/src/components/Form/Stepper.js]: null is already set as default

**Pull request closed without merging**